src/main/java/StringManipulator.java

```diff
@@ -0,0 +1,29 @@
+public class StringManipulator {
+    
+    public StringManipulator(){
+        // TODO: 02-Feb-21
+    }
+
+    public static String getFirstWord(String input){
+        return input.split(" ")[0];
+    }
+
+    public static String getStringAfterSlash(String input){
+        return input.substring(input.indexOf("/")+1+2);
+    }
+
+    public static String getStringAfterWhiteSpace(String input){
+        int firstWhiteSpaceIndex = input.indexOf(" ");
+        return input.substring(firstWhiteSpaceIndex+1);
+    }
+
+    public static String getStringAfterWhiteSpaceAndBeforeSlash(String input){
+        int firstWhiteSpaceIndex = input.indexOf(" ");
+        int firstSlashIndex = input.indexOf("/");
+        return input.substring(firstWhiteSpaceIndex+1, firstSlashIndex);
+    }
+
+    public static int getTaskNumberDone(String input){
+        return Integer.parseInt(input.substring(5));
+    }
+}
```

Comment:
Very good to create a new class for code with high reusability.

Reply:
Thank you! :)